aspnet/fundamentals/localization.rst

```diff
@@ -1,8 +1,336 @@
-.. include:: /../common/stub-topic.txt
+Globalization and Localization   
+==============================
 
-|stub-icon| Localization
-========================
+By `Rick Anderson`_, `Damien Bowden <https://twitter.com/damien_bod>`_, `Bart Calixto <https://twitter.com/bartmax>`_, and `Nadeem Afana <https://twitter.com/NadeemAfana>`__ 
 
-.. include:: /../common/stub-notice.txt
+Creating a multilingual website with ASP.NET Core will allow your site to reach a wider audience. ASP.NET Core provides services and middleware for localizing into different languages and cultures.
 
-.. _issue: https://github.com/aspnet/Docs/issues/653
\ No newline at end of file
+Internationalization involves `Globalization <https://msdn.microsoft.com/en-us/library/aa292081(v=vs.71).aspx>`__ and `Localization <https://msdn.microsoft.com/en-us/library/aa292137(v=vs.71).aspx>`__. Globalization is the process of designing apps that support different cultures. Globalization adds support for input, display, and output of a defined set of language scripts that relate to specific geographic areas.
+
+Localization is the process of adapting a globalized app, which you have already processed for localizability, to a particular culture/locale. The process of localizing your app also requires a basic understanding of relevant character sets commonly used in modern software development and an understanding of the issues associated with them. Although all computers store text as numbers (codes), different systems store the same text using different numbers. The localization process refers to translating the app user interface (UI) for a specific culture/locale. 
+
+`Localizability <https://msdn.microsoft.com/en-us/library/aa292135(v=vs.71).aspx>`__ is an intermediate process for verifying that a globalized app is ready for localization.
+
+The `RFC 4646 <https://www.ietf.org/rfc/rfc4646.txt>`__ format for the culture name is "<languagecode2>-<country/regioncode2>", where <languagecode2> is the language code and <country/regioncode2> is the subculture code. For example, ``es-CL`` for Spanish (Chile), ``en-US`` for English (United States), and ``en-AU`` for English (Australia). `RFC 4646 <https://www.ietf.org/rfc/rfc4646.txt>`__ is a combination of an ISO 639 two-letter lowercase culture code associated with a language and an ISO 3166 two-letter uppercase subculture code associated with a country or region.  See `Language Culture Name <https://msdn.microsoft.com/en-us/library/ee825488(v=cs.20).aspx>`__.
+
+Internationalization is often abbreviated to "I18N". The abbreviation takes the first and last letters and the number of letters between them, so 18 stands for the number of letters between the first "I" and the last "N". The same applies to Globalization (G11N), and Localization (L10N).
```

Comment:
@ryanbrandenburg  it's just just below in the Terms list:

- Internationalization (I18N): Describes both globalization and localization.